src/view/deposit-mobile.js

```diff
@@ -0,0 +1,100 @@
+import React from 'react';
+import { View, StyleSheet } from 'react-native';
+import { observer } from 'mobx-react';
+import PropTypes from 'prop-types';
+import { createStyles, maxWidth } from '../component/media-query';
+import { CopyText } from '../component/text';
+import { SplitBackground } from '../component/background';
+import MainContent from '../component/main-content';
+import { Header, Title } from '../component/header';
+import { CopyButton, Button, ButtonText } from '../component/button';
+import { CopiedNotification } from '../component/notification';
+import CopyDarkIcon from '../../src/asset/icon/copy-dark';
+import QRCode from '../component/qrcode';
+import { color, smallBreakWidth } from '../component/style';
+
+const baseStyles = {
+  qrWrapper: {
+    justifyContent: 'center',
+    alignItems: 'center',
+  },
+  qrCode: {
+    height: 170,
+    width: 170,
+  },
+  content: {
+    justifyContent: 'space-between',
+  },
+  copyTxt: {
+    textAlign: 'center',
+    maxWidth: 300,
+  },
+  btnWrapper: {
+    alignItems: 'center',
+  },
+  copyBtn: {
+    borderRadius: 0,
+    paddingLeft: 15,
+    paddingRight: 15,
+    backgroundColor: color.glas,
+  },
```

Comment:
We should create another `SmallCopyButton` component that we can reuse in the invoice screen.

Reply:
Sounds good! should i open an issue for that or did you want it in the same PR?